meta-flutter/README.md

```diff
@@ -39,5 +77,20 @@ You need to install libsystemd in the same way with the DRM-GBM backend.
 $ bitbake flutter-drm-eglstream-backend
 ```
 
+## Cross-building with Yocto SDK
+Installation of the cross-toolchain creates a cross-toolchain environment setup script in the directory that Yocto SDK was installed. Source the cross-toolchain environment setup file :
```

Comment:
Setup the cross-building toolchain environment using a script that you built and installed.